test/images/httpd/BASEIMAGE

```diff
@@ -1,7 +1,7 @@
-linux/amd64=httpd:2.4.38-alpine
-linux/arm=arm32v6/httpd:2.4.38-alpine
-linux/arm64=arm64v8/httpd:2.4.38-alpine
-linux/ppc64le=ppc64le/httpd:2.4.38-alpine
-linux/s390x=s390x/httpd:2.4.38-alpine
+linux/amd64=httpd:2.4.63-alpine
```

Comment:
httpd usually points to the `n-1` version to test some of the rolling update features of k8s, hence could you please use the `2.4.62` version instead?

Reply:
Addressed, thanks!